lib/xapi/implementation.rb

```diff
@@ -91,6 +91,8 @@ def assemble_readme
 #{readme_body}
 
 #{problem.source_markdown}
+
+#{incomplete_solutions_body}
```

Comment:
I think it's fine to put it at the bottom for now, in it's own section. It will draw people's eye for a while, and then they'll ignore it.